src/sinol_make/commands/run/__init__.py

```diff
@@ -42,20 +43,20 @@ def color_time(time, limit):
 
 
 def colorize_status(status):
-    if status == "OK": return util.bold(util.color_green(status))
-    if status == "  " or status == "??": return util.warning(status)
+    if status == Status.OK: return util.bold(util.color_green(status))
+    if status == Status.NO_STATUS or status == "??": return util.warning(status)
```

Comment:
What do you think of replacing `Status.NO_STATUS` with `Status.PENDING`? What does `status == '??'` mean?

Reply:
`PENDING` is a better name. The `??` is also present in original Kostka's code, but it wasn't used there either